plugin.video.worldstarhiphop/resources/lib/worldstarhiphop_list.py

```diff
@@ -94,21 +94,16 @@ def getVideos(self):
         # Parse response
```

Comment:
please drop the pyo compiled file

Reply:
@skipmodea1  When i look at the files, the pyo files are already not there?!

Reply:
nevermind, no idea how I missed them on the last submission. We should add this to the addon-checker